owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/OwnCloudClient.java

```diff
@@ -127,9 +121,6 @@ public int executeHttpMethod(HttpBaseMethod method) throws Exception {
 
     private void checkFirstRedirection(HttpBaseMethod method) {
         final String location = method.getResponseHeader(HttpConstants.LOCATION_HEADER_LOWER);
-        if (location != null && !location.isEmpty()) {
-            mRedirectedLocation = location;
-        }
     }
```

Comment:
Do we need checkFirstRedirection?

Reply:
AS suggested me to do this because of:

![Screenshot 2020-02-05 at 08 40 07](https://user-images.githubusercontent.com/15261363/73821069-6710ca00-47f3-11ea-934a-7dd1650cc6e5.png)